Simbody/tests/TestElasticFoundationForce.cpp

```diff
@@ -130,9 +130,146 @@ void testForces() {
     }
 }
 
+/**
+ * @brief This test compares the numerical result of a sphere
+ *        in contact with a plane, using the elastic foundation
+ *        model.
+ *        The analytical solution of this problem is given by
+ *        the product of the stiffness with the volume of
+ *        the sphere in the plane, i.e the volume of a spherical
+ *        cap.
+ *        The volume of a spherical cap is:
+ *          Vcap = Pi*h*h/3.0*(3.0*r-h)
+ *        where
+ *          r is the radis of the sphere
+ *          h the height of the cap. In our case, the penetration
+ *          depth
+ * @note If we want to go further, we can observe that doubling
+ *       the penetration depth results in multiplying the normal
+ *       effort by 4.
+ *       This is different from Hertz theory, where doubling the
+ *       penetration depth results in multiplying
+ *       the normal effort by 2^(3/2)~2.68
+ *
+ */
+void testEffSphereOnPlaneOldFormulation(bool verbose = false);
+void testEffSphereOnPlaneOldFormulation(bool verbose)
```

Comment:
You don't need to do this in two lines--you can put the default argument in the definition.